### ISSUE TITLE
Ruby Not able install  using rvm

### ISSUE THREAD

**Issue by rsurendraidexcel** [rvm/rvm#4776]
I am not able install ruby in ubutu 18.04  domain user 
/home/surendra@indexex.interal like folder i am facing problem 

![Screenshot from 2019-09-07 13-47-04](https://user-images.githubusercontent.com/7713535/64473194-df5f5a80-d183-11e9-9608-70a8ad451c68.png)

![Screenshot from 2019-09-07 13-32-56](https://user-images.githubusercontent.com/7713535/64473253-ee460d00-d183-11e9-8990-e0d49ed59f36.png)

**Comment by t12nslookup** [comment]
this looks like your RVM set-up is incorrect.

the path to your gemsets appears to be your homedirectory (rather than a subdirectory off of your home directory), and it's appending the gemset name at the end of your login name and trying to create a new directory ...

can you try removing and reinstalling RVM according to the install manual:

https://rvm.io/rvm/install

e.g.

curl -sSL https://get.rvm.io | bash -s stable --with-default-gems="rails haml"

**Comment by raelgc** [comment]
Another option: try the [Ubuntu dedicated package](https://github.com/rvm/ubuntu_rvm).

**Comment by dacat** [comment]
@raelgc Does this have anything to do with the fact his home directory is "/home/surendra@indexex.interal" which contains the '@' character? Is rvm parsing the install directory and stopping at the '@' ?  I just stumbled on this figured I mention what I saw.

**Comment by raelgc** [comment]
@dacat Indeed, it appears to be related to the username. Not sure if the dot or the @ sign.

Is it possible, in order to try to isolate the problem, create a new user with an username without any symbol, then add it to the `rvm` group, logout, login, and try to install a ruby?

**Comment by raelgc** [comment]
Sounds like a duplicate of #3790.
